### ISSUE TITLE
theHunter Call of the Wild: Graphical issues

### ISSUE THREAD

**Issue by el-christianito** [daniel-schuermann/mesa#168]
Hello

#### Summary
In theHunter: Call of the Wild, ACO compiler causes graphical issues. It looks like textures go missing close to oneself no matter the location. The problem also occurs on RADV_PERFTEST=llvmvs. It does not occur on RADV_PERFTEST=llvm.

#### How to reproduce
Run the game using RADV_PERFTEST=aco or llvmvs.

#### Screenshots
*ACO*
![theHunter_20191215215321_1](https://user-images.githubusercontent.com/9352289/70869032-ef5ab780-1f86-11ea-8c4f-ffd03412071b.jpg)
![theHunter_20191215215328_1](https://user-images.githubusercontent.com/9352289/70869033-f1bd1180-1f86-11ea-8be4-b33f16392269.jpg)

*LLVM*
![20191215215737_1](https://user-images.githubusercontent.com/9352289/70869035-fbdf1000-1f86-11ea-8ffe-6424d1a7a27f.jpg)
![20191215215751_1](https://user-images.githubusercontent.com/9352289/70869039-05687800-1f87-11ea-9850-6d4a04b98dec.jpg)

#### Specs
CPU Brand:  AMD Ryzen 5 2600X Six-Core 
RAM:  32132 Mb
Video Card:
    Driver:  X.Org Radeon RX 570 Series (POLARIS10, DRM 3.33.0, 5.3.12-1-MANJARO, LLVM 9.0.0)
    Driver Version:  4.5 (Compatibility Profile) Mesa 20.0.0-devel (git-3409c06e26)
    OpenGL Version: 4.5
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 144 Hz
    VendorID:  0x1002
    DeviceID:  0x67df
    Revision Not Detected
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1920 x 1080
    Desktop Resolution: 1920 x 1080
    Primary Display Size: 20.51" x 11.54" (23.50" diag)
                                            52.1cm x 29.3cm (59.7cm diag)
    Primary VRAM: 4096 MB

Operating System Version:
    "Manjaro Linux" (64 bit)
    Kernel Name:  Linux
    Kernel Version:  5.3.12-1-MANJARO
    X Server Vendor:  The X.Org Foundation
    X Server Release:  12005000
    X Window Manager:  KWin
    Steam Runtime Version:  <Runtime disabled> (issue happens using current Steam Runtime too)

**Comment by daniel-schuermann** [comment]
Hi @el-christianito, thx for reporting this issue. Would you be able to create a renderdoc capture of the scene?

**Comment by el-christianito** [comment]
Hi @daniel-schuermann 

I was giving it a try, but the game crashes whenever I'm trying to make a renderdoc capture. Is there a common way to prevent this?

What I found out is that the missing textures are related to tesselation settings. The higher I set tesselation, the further from the player textures are missing.

Ultra:
![theHunter_tesselationUltra](https://user-images.githubusercontent.com/9352289/71312937-dabc6a80-2431-11ea-9ab8-68a9faf47698.jpg)

Medium:
![theHunter_tesselationMed](https://user-images.githubusercontent.com/9352289/71312939-df811e80-2431-11ea-8cb9-532b45b17755.jpg)

Low:
![theHunter_tesselationLow](https://user-images.githubusercontent.com/9352289/71312940-e1e37880-2431-11ea-843f-97ea9038542e.jpg)

It's not completely gone on low tesselation though. There are still some missing spots once in awhile.
![theHunter_tesselationLow_issue](https://user-images.githubusercontent.com/9352289/71312947-e445d280-2431-11ea-9eb6-c66d7cb8fdaa.jpg)
